Service/Endpoint/EndpointFactoryInterface.php

```diff
@@ -13,6 +13,7 @@ interface EndpointFactoryInterface
         EndpointInterface::METHOD_PATCH,
         EndpointInterface::METHOD_PUT,
         EndpointInterface::METHOD_DELETE,
+        EndpointInterface::METHOD_HEAD,
```

Comment:
What do you think about adding validation to endpoints with `HEAD` method to not to have Response DTO?